csunplugged/topics/management/commands/_CurriculumAreasLoader.py

```diff
@@ -56,10 +56,19 @@ def load(self):
                     "Curriculum Area"
                 )
 
+            curriculum_area_colour = curriculum_area_data.get("colour", None)
+            if curriculum_area_colour is None:
+                raise MissingRequiredFieldError(
+                    self.structure_file_path,
+                    ["colour"],
```

Comment:
we could and probably should, when I first started implementing it I did it this way so as to not get duplicate code everywhere, but can probably find a way to minimise that now. Maybe use the full list here and make an issue to go and change that error (and all calls to it) later?